plugins/kubernetes-react/src/components/Cluster/Cluster.tsx

```diff
@@ -89,13 +92,17 @@ const ClusterSummary = ({
         spacing={0}
       >
         <Grid item>
-          <StatusOK>{totalNumberOfPods} pods</StatusOK>
+          <StatusOK>
+            {totalNumberOfPods} {t('cluster.pods')}
+          </StatusOK>
         </Grid>
         <Grid item>
           {numberOfPodsWithErrors > 0 ? (
-            <StatusError>{numberOfPodsWithErrors} pods with errors</StatusError>
+            <StatusError>
+              {numberOfPodsWithErrors} {t('cluster.podsWithErrors')}
```

Comment:
Not sure if this make sense, I think this should just be a single label

```suggestion
              {t('cluster.podsWithErrors', { count: numberOfPodsWithErrors })}
```